script/utils/rewards_testing/TestRewardsV2.sol

```diff
@@ -0,0 +1,334 @@
+        strategies[5] = IStrategy(0xdcCF401fD121d8C542E96BC1d0078884422aFAD2);
+
+        return strategies;
+    }
+
+    function _getAVStrategies() internal pure returns (IStrategy[] memory) {
+        IStrategy[] memory strategies = new IStrategy[](15);
+
+        strategies[0] = IStrategy(0x08f8544E61Ebfa22e7c6ef9af9eFd428091b27AF);
+        strategies[1] = IStrategy(0x31741340ab31e90f0624d2C843B323a97755f43a);
+        strategies[2] = IStrategy(0x845aDA440F3BA5652e6f7c6Bbcd8fCAd068Be9c9);
+        strategies[3] = IStrategy(0xcD6EDf68a5Fc79BBc14A0eCa1C8bbDe495fB1a5f);
+        strategies[4] = IStrategy(0x77335a08a877cd874165bDC766feeD951a0d84c8);
+        strategies[5] = IStrategy(0x53C916338FcA4f5541bc2eA5E5f7F0Ca6f0dAf6a);
+        strategies[6] = IStrategy(0x77335a08a877cd874165bDC766feeD951a0d84c8);
+        strategies[7] = IStrategy(0x51366AEf35475e82B864f2E7867B293c3BC61D86);
+        strategies[8] = IStrategy(0x5803f7D0D273aa0B9774C0aDAB98A23ec348Ea77);
+        strategies[9] = IStrategy(0xe5e8BcBd1DDD07460fA67f617a2D018ce0f5b7bf);
+        strategies[10] = IStrategy(0x419B6Ba569169826d9A45565E651F03B8DdfA332);
+        strategies[11] = IStrategy(0x9b5F92ed40e1436Ac8E31fd7f7bc6083edBd5E71);
+        strategies[12] = IStrategy(0xE34b1Da874E6B9f091540c850095b99B4D3B475D);
+        strategies[13] = IStrategy(0x8EF42562E9F1b81010a54615673739dd91E10016);
+        strategies[14] = IStrategy(0xdE2788cb747b51a1747Bd59342b6214652Ddeb10);
+
+        // sort strategies by address
+        for (uint256 i = 0; i < strategies.length; i++) {
+            for (uint256 j = i + 1; j < strategies.length; j++) {
+                if (address(strategies[i]) > address(strategies[j])) {
+                    IStrategy temp = strategies[i];
+                    strategies[i] = strategies[j];
+                    strategies[j] = temp;
+                }
+            }
+        }
+
+        return strategies;
+    }
+
+    function _getAllRegisteredOperators() internal returns(address[] memory, uint256[] memory){
+        address[] memory operators = new address[](maxOperatorIndex);
+        uint256[] memory privateKeys = new uint256[](maxOperatorIndex);
+        for(uint256 i = 0; i < operators.length; i++){
+            (address operator, uint256 privateKey) = deriveRememberKey(MNEMONIC, uint32(i));
+            operators[i] = operator;
+            privateKeys[i] = privateKey;
+        }
+        return (operators, privateKeys);
+    }
+
+    function _getRandomOperatorRewards(address[] memory operators, uint256 max) internal returns (IRewardsCoordinator.OperatorReward[] memory) {
+        IRewardsCoordinator.OperatorReward[] memory operatorRewards = new IRewardsCoordinator.OperatorReward[](operators.length);
+        for (uint256 i = 0; i < operators.length; i++) {
+            operatorRewards[i] = IRewardsCoordinator.OperatorReward({
+                operator: operators[i],
+                amount: vm.randomUint(1, max)
```

Comment:
This is probably going to fail. Because we now have the sum of all operator amounts < `MAX_REWARDS_AMOUNT`

Reply:
So we should cap the max at like `1e35-1` (since we have 1000 operators right?)

Reply:
we're only doing 80 operators and max is set to 1e18 above